contexts/ed25519-signature-2018-v1.json

```diff
@@ -0,0 +1,86 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "@protected": true,
+
+    "Ed25519VerificationKey2018": {
+      "@id": "https://w3id.org/security#Ed25519VerificationKey2018",
+      "@context": {
+        "@protected": true,
+        "id": "@id",
+        "type": "@type",
+        "controller": {
+          "@id": "https://w3id.org/security#controller",
+          "@type": "@id"
+        },
+        "publicKeyBase58": {
+          "@id": "https://w3id.org/security#publicKeyBase58"
+        },
+        "privateKeyBase58": {
+          "@id": "https://w3id.org/security#privateKeyBase58"
+        }
```

Comment:
I thought we weren't defining this to ensure that people wouldn't accidentally publish private keys in documents (dropped during compaction). This would violate that and open up a new security attack surface.

Reply:
Ahhh yeah good call. I'll remove it, just in case.